app/models/form_submission.rb

```diff
@@ -10,6 +10,47 @@ class FormSubmission < ApplicationRecord
 
   validates :form_type, presence: true
 
+  class << self
+    # rubocop:disable Metrics/MethodLength
+    def with_latest_intake(user_account)
```

Comment:
I might rename this `with_latest_benefits_intake_uuid`. I don't think `intake` is a noun on its own that is understood in this context. The API's full name is "Benefits Intake API" so I could imagine a `Benefits Intake` being a thing but... well, I'm not sure what that would mean exactly either tbh! 😄 I know my alternative is much more verbose but I think/hope it'll make more sense going forward.

Reply:
I'm good with that 👍 .  I'll change it.